Source/ULibs.Win32/README.md

```diff
@@ -0,0 +1 @@
+﻿Contains various Win32 imports. We typically use [PInvoke.net](https://pinvoke.net) to grab Win32 exports as needed, but this can result in the same structure being used in multiple places. The point of this library is to remove this duplication.
```

Comment:
Ah, I've also just noticed that all the types in this library are public, and one of the stated purposes here is about avoiding duplication. This is contrary to this repo's README:

> Micro-libraries are independent and internal to each project that references them. You don't need to worry about version conflicts when you combine code that may use different versions of the library.

One of the original aims was to keep micro-libraries small and internal, which encourages duplication where necessary in preference to avoiding package version conflicts. That relies on keeping the source internal rather than public, though.

Reply:
That is a good spot.

There's two things I should do here - I should make all the types `internal` and I should fix the readme to be clearer.